CONTRIBUTING.rst

```diff
@@ -6,6 +6,34 @@ help with implementing changes in Anaconda, please follow our
 `blog series <https://rhinstaller.wordpress.com/2019/10/11/anaconda-debugging-and-testing-part-1/>`_ or
 an `addon guide <http://rhinstaller.github.io/anaconda-addon-development-guide/index.html>`_ to create Anaconda addon.
 
+How to run make commands
+------------------------
+
+Anaconda has plenty of dependencies and because of that it's hard to set environment to
+with Anaconda properly. To get all the dependencies you are free to use script in the Anaconda
+repository. There are two ways how to approach.
+
+a) I want to have my host clean and run autotools from a container:
+
+``sudo dnf install toolbox``
+``toolbox create``
+``toolbox enter``
```

Comment:
Please make this a single pre-formatted paragraph, like this it [renders wrongly](https://github.com/rhinstaller/anaconda/blob/0322be290e02b489eb71e9b52c5b03ca2215d9be/CONTRIBUTING.rst).